lib/dragonfly_libvips/processors/encode.rb

```diff
@@ -38,10 +38,10 @@ def call(content, format, options = {})
         output_options.delete("Q") unless /jpg|jpeg/i.match?(format.to_s)
         output_options["format"] ||= format.to_s if /gif|bmp/i.match?(format.to_s)
 
-        img = ::Vips::Image.new_from_file(content.path, DragonflyLibvips.symbolize_keys(input_options))
+        img = ::Vips::Image.new_from_file(content.path, **DragonflyLibvips.symbolize_keys(input_options))
 
         content.update(
-          img.write_to_buffer(".#{format}", DragonflyLibvips.symbolize_keys(output_options)),
+          img.write_to_buffer(".#{format}", **DragonflyLibvips.symbolize_keys(output_options)),
```

Comment:
definition of `write_to_buffer` - https://github.com/libvips/ruby-vips/blob/e705f88ed7b071ff5065ff63f4b757251a16b074/lib/vips/image.rb#L622